.github/workflows/deploy.yml

```diff
@@ -10,11 +10,27 @@ jobs:
     deploy:
         name: Deploy
         runs-on: ubuntu-latest
+        env:
+          DISCORD_TOKEN: ${{ secrets.DISCORD_TOKEN }}
+          DISCORD_CLIENT_ID: ${{ secrets.DISCORD_CLIENT_ID }}
         steps:
             - name: Checkout
               uses: actions/checkout@v3
 
+            - name: Setup PNPM
+              uses: pnpm/action-setup@v2
+
+            - name: Setup node@20
+              uses: actions/setup-node@v3
+              with:
+                node-version: 20
+                cache: "pnpm"
+
+            - name: Install dependencies
+              run: pnpm install
```

Comment:
I guess we do need this for the `preCommand`, correct?
IIRC the wrangler build pipeline during deploy does install deps automatically, so maybe worth double checking.

Reply:
We do need this. Install deps might be automatic, but the rest is not so I'd like to keep it like this. Your initial script actually failed because pnpm was not installed

Reply:
Interesting. I do have workflows without it and they work. But it's fine to keep them!